src/main/java/frc/robot/subsystems/GrabberSubsystem.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.util.armPorts;
+
```

Comment:
Unused import, along with with incorrect case for armPorts. Should be "ArmPorts"